bin/cli.js

```diff
@@ -70,6 +70,22 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			// When webpack is not installed and no args passed to the CLI
```

Comment:
I think these lines speak for themselves

Reply:
??

Reply:
Remove the comment, code is clear enough